app/views/api/applications/index.html.slim

```diff
@@ -1,4 +1,4 @@
 h1 Applications
-= pf_button_in_title 'Create Application', new_admin_service_application_path(@service)
+= pf_button_in_title 'Create Application', new_admin_service_application_path(@service) unless site_account.master?
```

Comment:
Not sure this should be part of this pr as it has nothing to do with the typeahead filter? Also, the problem of with the new application page where a drop down of accounts causes page time outs could occur for any provider as well if they have a large amount of developer accounts, no?

Reply:
Thanks for spotting this